components/Collections/src/MDCCollectionViewEditing.h

```diff
@@ -55,4 +55,15 @@
  */
 - (void)setEditing:(BOOL)editing animated:(BOOL)animated;
 
+/**
+ Updates the position of the cell being reordered to match the user's touch location. This method
+ is a no-op if a cell is not currently being reordered.
+ 
+ While the reordering cell's location is updated automatically to match the user's touch location,
+ this method allows for the explicit update of the cell's position which may be required when the
+ cell's location changes due to a layout update (such as performing collection view changes while
+ the user is reordering).
+ */
+- (void)updateReorderCellPosition;
```

Comment:
Please also answer the following questions in the docs:

- When is this API typically meant to used?
- From where is this API typically meant to be used?

E.g. "Call foo from your Bar's doLayout implementation."